lldb/bindings/interface/SBFrameExtensions.i

```diff
@@ -44,6 +44,16 @@ STRING_EXTENSION_OUTSIDE(SBFrame)
                 def __init__(self, regs):
                     self.regs = regs
 
+                def __iter__(self):
+                    return self.get_registers()
+
+                def get_registers(self):
+                    for i in range(0,len(self.regs)):
+                        rs = self.regs[i]
+                        for j in range (0,rs.num_children):
```

Comment:
The `range` expressions in this function can be simplified. Dropping the 0 means the list of numbers automatically starts at 0:
```
for i in range(len(self.regs)):
...
for j in range(rs.num_children):
```

Reply:
Unless there's some difference in behavior, I don't mind burning a few characters to make the range explicit.

Reply:
Fair, explicit is good. I meant it as a suggestion originally.